doc/authors.md

```diff
@@ -17,12 +17,12 @@ Cory Fields (35)
 mdr0id (22)
 Larry Ruane (16)
 syd (15)
+Jonathan "Duke" Leto (15)
 Matt Corallo (13)
-Jonathan "Duke" Leto (12)
+Ariel Gabizon (11)
```

Comment:
Similarly, `Ariel Gabizon` now has 11 commits, when previously `Ariel Gabizon` and `Ariel` collectively had 12 commits. Off-by-one error?

Reply:
Don't know, automated script handles this.